sdk/python/kfp/components/executor_test.py

```diff
@@ -77,33 +81,32 @@
     },
     "parameters": {
       "output_parameter_path": {
-        "outputFile": "gs://some-bucket/some_task/nested/output_parameter"
+        "outputFile": "%(s)s/gcs/some-bucket/some_task/nested/output_parameter"
```

Comment:
`s` isn't very readable, should we name it to something else? `test_dir`, `root_dir`, or something else?

Reply:
Definitely -- thanks for the close reviews, @chensun